packages/doctor/lib/general.js

```diff
@@ -5,9 +5,21 @@ import NodeDetector from './node-detector';
 import {util} from '@appium/support';
 import {EOL} from 'os';
 import '@colors/colors';
+import {resolveAppiumHome} from '@appium/support/build/lib/env';
```

Comment:
```suggestion
import {env} from '@appium/support';
const {resolveAppiumHome} = env;
```

that's the public API; you don't want to rely on the structure of the `build` folder, which is an implementation detail

Reply:
or `await env.resolveAppiumHome()`

Reply:
oh, true. I just did auto-import via VSCode, thanks